test/org/sagebionetworks/bridge/dynamodb/DynamoFPHSExternalIdentifierDaoTest.java

```diff
@@ -0,0 +1,133 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.List;
+
+import javax.annotation.Resource;
+
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.sagebionetworks.bridge.exceptions.EntityAlreadyExistsException;
+import org.sagebionetworks.bridge.exceptions.EntityNotFoundException;
+import org.sagebionetworks.bridge.models.accounts.ExternalIdentifier;
+import org.sagebionetworks.bridge.models.accounts.FPHSExternalIdentifier;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import com.google.common.collect.Lists;
+
+@ContextConfiguration("classpath:test-context.xml")
+@RunWith(SpringJUnit4ClassRunner.class)
+public class DynamoFPHSExternalIdentifierDaoTest {
+
+    @Resource
+    private DynamoFPHSExternalIdentifierDao dao;
+    
+    @BeforeClass
+    public static void beforeClass() {
+        DynamoInitializer.init(DynamoFPHSExternalIdentifier.class);
+    }
+
+    @After
+    public void after() {
+        dao.deleteAll();
+    }
+    
+    @Test
+    public void verifyExternalIdDoesNotExist() throws Exception {
+        ExternalIdentifier externalId = new ExternalIdentifier("abc");
```

Comment:
It might be worthwhile generating a random prefix to the external IDs for this test. This way, if for some reason a previous test run fails to cleanup, or if a dev is locally testing external IDs and just happens to use the same external ID, this won't cause spurious failures.